gcp/deployment-packages/autoscale-byol/README.md

```diff
@@ -0,0 +1,130 @@
+# GCP Deployment Manager package for Check Point Autoscaling BYOL solution
+This directory contains CloudGuard IaaS deployment package for Check Point Autoscaling (BYOL) solution published in the [GCP Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/checkpoint.vsec?tab=Overview).
+
+# How to deploy the package manually
+To deploy the Deployment Manager's package manually, without using the GCP Marketplace, follow these instructions:
+1. [Download](https://github.com/CheckPointSW/CloudGuardIaaS/archive/master.zip) the CloudGuardIaaS repository
+2. Browse to the CloudGuardIaaS/gcp/deployment-packages/autoscale-byol/ directory
+3. Fill variables in the test_config.yaml(see below for variables descriptions).
+4. Log in into [Google Cloud Platform](https://console.cloud.google.com)
+5. [Activate Cloud Shell](https://cloud.google.com/shell/docs/using-cloud-shell)
+6. Set your Cloud Platform project in the Cloud Shell session use:
+        
+        gcloud config set project [PROJECT_ID]
+7. Upload content of the CloudGuardIaaS/gcp/deployment-packages/autoscale-byol/ directory  to the [cloud shell](https://cloud.google.com/shell/docs/uploading-and-downloading-files) 
+8. Launch deployment by running:
+        
+         gcloud deployment-manager deployments create [DEPLOYMENT_NAME] --config test_config.yaml
+9. Make sure the deployment finished successfully. <br>Example of successful deployment output:
+        
+        The fingerprint of the deployment is h8R2exQYuc4bzlO14boUhg==
+        Waiting for create [operation-1585217870871-5a1bf4c15b9ea-f8d824d5-1089cc78]...done.
+        Create operation operation-1585217870871-5a1bf4c15b9ea-f8d824d5-1089cc78 completed successfully.
+        NAME                           TYPE                                   STATE      ERRORS  INTENT
+        mig-as                         compute.v1.regionAutoscaler            COMPLETED  []
+        mig-igm                        compute.v1.regionInstanceGroupManager  COMPLETED  []
+        mig-vpc-icmp                   compute.v1.firewall                    COMPLETED  []
+        mig-vpc-udp                    compute.v1.firewall                    COMPLETED  []
+        mig-tmplt                      compute.v1.instanceTemplate            COMPLETED  []
+
+## test_config.yaml variables
```

Comment:
why is it called test_config if it's the actual variables for the deployment? is this the way it's called in the markteplace templates?